cmd/snap-update-ns/bootstrap.c

```diff
@@ -46,13 +46,20 @@ int bootstrap_errno = 0;
 const char *bootstrap_msg = NULL;
 
 // setns_into_snap switches mount namespace into that of a given snap.
-static int setns_into_snap(const char *snap_name)
+// If the uid argument is non-zero the mount namespace of a specific user is joined.
+// Otherwise the system-wide mount namespace of the provided snap is used.
```

Comment:
I suggest:
```
// If the uid argument is non-zero the mount namespace of the provided snap
// for the specific user is joined. Otherwise the system-wide mount namespace
// of the provided snap is used.
```